Source/ACE.Server/Entity/Cloak.cs

```diff
@@ -19,7 +19,7 @@ public static bool TryProcSpell(Creature defender, WorldObject attacker, WorldOb
         {
             if (cloak == null) return false;
 
-            if (!RollProc(damage_percent))
+            if (!RollProc(damage_percent, cloak.ItemMaxLevel.Value))
```

Comment:
this will throw a null pointer exception if a cloak doesn't have a level

i will adjust the code in this pr around a bit later today